reporters/kamon-prometheus/src/main/resources/reference.conf

```diff
@@ -52,6 +52,16 @@ kamon.prometheus {
       1048576
     ]
 
+    percentage-buckets = [
```

Comment:
It saddens me that there is such a wide spread across those buckets but well, that's how it has to be :shrug: